src/cacheService.js

```diff
@@ -64,12 +64,14 @@ class CacheService {
   async getThread (req, res, next) {
     const spaceName = req.query.space
```

Comment:
want to also allow API to get thread with full address? and offer both full address or space/name/mod/members

Reply:
Oh, didn't think about that. I can add it yeah!